manifests/server/binarylog.pp

```diff
@@ -3,19 +3,22 @@
 
   $options = $mysql::server::options
   $includedir = $mysql::server::includedir
+  $manage_binlog_dir = $mysql::server::manage_binlog_dir
 
   $logbin = pick($options['mysqld']['log-bin'], $options['mysqld']['log_bin'], false)
 
-  if $logbin {
-    $logbindir = mysql_dirname($logbin)
+  if $manage_binlog_dir == true {
+    if $logbin {
+      $logbindir = mysql_dirname($logbin)
```

Comment:
Since this is just an internal variable name, could you make it `$binlog_dir` to match the `$manage_binlog_dir` name?